app/Models/Database.php

```diff
@@ -51,6 +51,7 @@ class Database extends Model
         'database_host_id' => 'required|exists:database_hosts,id',
         'database' => 'required|string|alpha_dash|between:3,100',
         'username' => 'string|alpha_dash|between:3,100',
+        'max_connections' => 'string',
```

Comment:
Is there a reason why you are using `string` instead of `sometimes|integer` or `nullable|integer` as the validation rule?

Reply:
As referred to my last comment, I made a commit hours ago and it was changed to `nullable`

https://github.com/Sir3lit/panel/blob/82dd7dc8e3b913839685728fb41f34139ee1d871/app/Models/Database.php#L54